cypress/integration/homepage.js

```diff
@@ -14,4 +14,34 @@ describe('Homepage', () => {
         expect(value.find('a').attr('href')).to.include('/locations/');
       });
   });
+
+  it('should open an article when clicking it', () => {
+    cy.get('.grav-c-card-basic')
+      .find('a')
+      .first()
+      .click();
```

Comment:
Question: Does this not need some kind of assertion at the end?

Reply:
Yes, I think there should be something to assert that the page exists and we've navigated to it.